shell/README.md

```diff
@@ -32,7 +32,16 @@ and then add the directory to your `$fpath` in your `.zshrc`, `.zsh_profile` or
 
 #### Oh my Zsh
 
-If you are using the popular [oh-my-zsh](https://github.com/robbyrussell/oh-my-zsh) framework to manage your zsh plugins, move the file `exercism_completion.zsh` into `~/.oh-my-zsh/custom`.
+If you are using the popular [Oh My
+Zsh](https://github.com/robbyrussell/oh-my-zsh) framework to manage your zsh
+plugins, you need to move the file `exercism_completion.zsh` to a new custom
+plugin:
+
+    mkdir -p $ZSH_CUSTOM/plugins/exercism
+    cp ../shell/exercism_completion.zsh $ZSH_CUSTOM/plugins/exercism/_exercism
+
+Then edit the file `~/.zshrc` to include `exercism` in the list of plugins.
+Completions will be activated the next time you open a new shell.
```

Comment:
```suggestion
Completions will be activated the next time you open a new shell.
If the completions doesn't work, you should update Oh my Zsh to the latest version with `upgrade_oh_my_zsh`. Oh my zsh now checks whether the plugin list has changed (more accurately, `$fpath`) and resets the zcompdump file.
```

Reply:
I think it is "Oh My Zsh" as a name, so we should use proper title for referencing.

Reply:
"If the completions do not work, you…"

Reply:
Thanks for the review mate :) I've edited the suggestion.

Reply:
I've added this text to the latest commit, except I replaced `upgrade_oh_my_zsh` to the new command, `omz update`.